.appveyor.yml

```diff
@@ -32,7 +32,14 @@ build:
   parallel: true
 
 before_build:
-  - git submodule update --init --recursive
+  - curl -sS https://gitlab.com/illwieckz/git-checkout-modules/raw/master/git-checkout-modules -o git-checkout-modules
+  # The git-checkout-modules command is normally equivalent to 'git submodule update --init --recursive',
```

Comment:
`git-checkout-modules --update` is equivalent to `git submodule update --init --recursive`, not `git-checkout-modules` alone.

Reply:
With the 'git-checkout-modules command' I am referring to the entire 5 lines. Maybe I will s/command/command below/.